### ISSUE TITLE
Kevlar vest thread cost too high

### ISSUE THREAD

**Issue by Warfoniusz** [CleverRaven/Cataclysm-DDA#66071]
### Describe the bug

Regular Kevlar Vest crafting requires 696 of appropriate thread (kevlar, nomex or heavy duty), while the XL version requires 300.

### Attach save file

N/A

### Steps to reproduce

1. Check the recipe cost for thread of Kevlar Vest and XL Kevlar Vest

### Expected behavior

Swap the thread cost between these 2 vests/reduce the thread cost of the regular vest.

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.2965 (22H2)
- Game Version: cb91dd7 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions],
    Blaze Industries [blazeindustries],
    Magiclysm [magiclysm],
    DinoMod [DinoMod],
    Bombastic Perks [bombastic_perks],
    Mythical Martial Arts [MMA],
    Translate Complex Dialogue [translate_dialogue],
    Stats Through Kills [stats_through_kills],
    Stats Through Skills [StatsThroughSkills]
]

**Comment by RenechCDDA** [comment]
Just needs someone to figure out what the value is supposed to be and edit the number. (Seems the XL is too low, rather than the other way around)

https://github.com/CleverRaven/Cataclysm-DDA/blob/4f113f720614bf82dfa7f49c6c29b2067d05a4a2/data/json/recipes/armor/torso.json#L2099

armor_kevlar_layered 30 ---> armor_kevlar_layered [some number]

@anoobindisguise If you could weigh in on how much kevlar the XL should take?

**Comment by randomtyper** [comment]
Huh, the XL Kevlar vest is entirely different in terms of materials, stats and thicknesses from the light Kevlar vest. The disassembly and crafting materials differ too: XL Kevlar is made out of layered Kevlar, while Kevlar vest is made out of normal kevlar sheets.

See:

https://cdda-guide.nornagon.net/item/xlkevlar
https://cdda-guide.nornagon.net/item/kevlar
https://cdda-guide.nornagon.net/item/level_3_vest
https://cdda-guide.nornagon.net/item/ballistic_vest_esapi

Linked lv 3 ballistic vests, as those are much closer in terms of stats to XL Kevlar (yet have MUCH lower yields on disassembly despite being of same thickness and size modifier for material being 1.25-1.5).

I'd say rename XL Kevlar vest into handmade mutant ballistic vest, normalise thread requirements to Kevlar vest grade + multiplier, lower mutie vest panel requirements (30 layered panels is A LOT of Kevlar, 20 would be much more reasonable considering the thickness and multiplier for body size),

**Comment by randomtyper** [comment]
Huh, yeah, I think I know why the thread requirement is so low: layered Kevlar is supposed to have much less stiching than unlayered (the Kevlar is pre-layered, after all) per the material "crafting set". Kinda simplifies yet complicates things. As such, XL Kevlar should be treated as completely different item from the normal Kevlar vest. Maybe add an actual light mutie vest (true XL Kevlar vest) and a handmade layered Kevlar vest to fill out the gaps.

**Comment by randomtyper** [comment]
Of course, we could axe the XL Kevlar vest alltogether, but it would essentially leave large mutants without Kevlar torso armor options, and it would be a bit boring. The mutie vest sacrifices quite a lot ahyways, compared to US army ballistic vest.

**Comment by EternalLaughter** [comment]
This seemed to have been fixed at one point, and looks like it can be closed.

<img width="1246" height="886" alt="Image" src="https://github.com/user-attachments/assets/ae6aeba8-d2d4-44f2-8d8e-769b7e27b885" />

<img width="1246" height="886" alt="Image" src="https://github.com/user-attachments/assets/72314763-6ca4-4ca9-a1b3-24ee9274549a" />

**Comment by RenechCDDA** [comment]
Sounds good to me.
